.vscode/extensions.json

```diff
@@ -0,0 +1,14 @@
+{
+  "recommendations": [
+    "mikestead.dotenv",
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
+    "eamodio.gitlens",
```

Comment:
I had a comment between these 2 lines and I remembered you can't leave comment sin .json files. The first 3 are the only ones that are really needed, the rest are just my personal recommendations.